### DISCUSSION TITLE
Embed Media on Behance

### DISCUSSION THREAD

**Discussion by nanamasilo** [cables-gl/cables_docs#476] [! Welcome - readme first!]
Hi, I've just messaged to Behance support because apparently it looks like they do not support your website as a source of embed media. It sais the code is invalid. One and a half year ago I had no issue with just copy-pasting the embed so it is current I think. (My patch is public, so it is not the case)
I just wanted to make sure that it is not a recent issue inside cables.
If not, would you be so cool and kind to consider reaching out to Behance to support your tool this way? (I am not sure if my voice is enough)

Thank you in avance!
![Képernyőfelvétel (547)](https://user-images.githubusercontent.com/123971316/215524391-0b5ef30f-d0df-4d69-8025-9b6322b74dbe.png)

**Comment by steam0r** [comment]
hey,

fromt the looks of it is seems that they have a whitelist of things to be allowed to embed. sorry, we can't provide support for integration into any platform. feel free to contact them and if they have any technical complaints about our integration (which is just a normal iframe after all) post them here and we will see what we can do.

i do not know "behance" at all, but you might have better chances with any of the other export options and then uploading that to their platform if possible. again: sorry, we simply do not have the ressources to work with all platforms for a working integration.

best,
stephan/steam
